db/seeds.rb

```diff
@@ -39,6 +47,8 @@
   la.email_address = "planning@buckinghamshire.gov.uk"
   la.feedback_email = "planning.digital@buckinghamshire.gov.uk"
   la.press_notice_email = "planning.digital@buckinghamshire.gov.uk"
+  la.email_reply_to_id = "550e8400-e29b-41d4-a716-446655440000"
+  la.active = true
 end
 
 admin_roles = %i[assessor reviewer administrator]
```

Comment:
I think is better to move [this](https://github.com/unboxed/bops/blob/main/app/services/letter_sending_service.rb#L56-L70) logic on [local_autohity model](https://github.com/unboxed/bops/blob/main/app/models/local_authority.rb). By doing that we won't need to update it here.